lib/matplotlib/backend_bases.py

```diff
@@ -2995,6 +2995,59 @@ def _zoom_pan_handler(self, event):
             elif event.name == "button_release_event":
                 self.release_zoom(event)
 
+    def _start_event_axes_interaction(self, event, method="zoom"):
```

Comment:
Never called without an argument, so make it keyword-only.
```suggestion
    def _start_event_axes_interaction(self, event, *, method):
```